example-data.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Please link the API spec you matched. It's not otherwise noted in this repo.
https://jsonapi.org/format/